packages/distribution/addon/components/cd-navigation/section.js

```diff
@@ -1,10 +1,42 @@
 import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
 import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
 
+function findInquiryParams(routeInfo) {
+  if (
+    routeInfo.paramNames.includes("from") &&
+    routeInfo.paramNames.includes("to")
+  ) {
+    return routeInfo.params;
+  }
+
+  return findInquiryParams(routeInfo.parent);
+}
+
 export default class CdNavigationSectionComponent extends Component {
+  @service router;
+
   @tracked expanded = true;
 
+  get isActive() {
+    try {
+      const { from = null, to = null } = findInquiryParams(
+        this.router.externalRouter.recognize(this.router.currentURL)
```

Comment:
For better understanding: Why is this approach needed instead of using currentRoute?

Reply:
Good question.. I found an even simpler way to solve this. I now used the `isActive` method of the router to check the active state of the sections inquiries.